app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -295,7 +295,7 @@ def pub_img_os(image_id)
   end
 
   def normalize_os(os_name)
-    os_name.sub!("red-", "redhat-") if os_name.start_with?("red-")
+    os_name.sub!("red-", "redhat-") if os_name&.start_with?("red-")
```

Comment:
Won't this just fail later if `os_name` is nil:
```
>> OperatingSystem.normalize_os_name(nil)
Traceback (most recent call last):
        1: from app/models/operating_system.rb:76:in `normalize_os_name'
NoMethodError (undefined method `downcase' for nil:NilClass)
```

Lets either do `normalize_os(os_name) if os_name` in the callers or `return if os_name.nil?` here